### ISSUE TITLE
Reconnecting to an old session discard some settings

### ISSUE THREAD

**Issue by alekspickle** [zellij-org/zellij#890]
**Basic information**
`zellij --version`: zellij 0.20.0
`tput lines`: 58
`tput cols`: 123
`uname -av` or `ver`(Windows): Linux pickle 5.11.0-37-generic #41~20.04.2-Ubuntu SMP Fri Sep 24 09:06:38 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

List of programs you interact with as, `PROGRAM --version`: output cropped meaningful, for example:
`cargo 1.56.0 (4ed5d137b 2021-10-04)`
`rustc 1.56.0 (09c42c458 2021-10-18)`
`git version 2.33.0`
`zsh 5.8 (x86_64-ubuntu-linux-gnu)`

**Further information**
I generally tend to keep one session always and this seems to work buggy with `zellij`. 
The issue I'm having is after some time options that I loaded `zellij` with from my config file seem to fall out.
For example I disable mouse functionality but it turned on after 3-4 reconnection (or it is a matter of time, hard to tell).

I guess it would be cool to reload session without shutting it down?

**Comment by a-kenji** [comment]
The client is currently responsible for some settings.
You should be able to disable the mouse-mode on connecting to the session.
Or you could disable it in the configuration file, that way it is picked up for every client.

**Comment by alekspickle** [comment]
Yeah, I know that there is such option, the issue is exactly for the case I used it and the long-running session lost it somehow...
I did not know that mouse flag also works for the attach though! It solves my problem then :

not sure whether to close issue. I generally feel that some reload session with shutting it down and opening it like it were before would be useful feature?..

**Comment by a-kenji** [comment]
I am also not too sure, I guess we can close it for now?

The way I see it:
This is an option that is important for the client, so the client can set it.
The server doesn't care if the client wants `mouse_mode` on, or off.

On attaching to a running session the client looks what you want from it:
- it checks the configuration if you want the mouse mode on, or off
- it checks the options if you want the mouse mode on, or off

Depending on what settings you want the attaching client to have, it will have.

I am happy to open it again, or discuss it in a different issue.
If there is more input.

>  I generally feel that some reload session with shutting it down and opening it like it were before would be useful feature?

I agree. That will be more closely related to  #575 I think.

Thanks for reaching out! Please don't hesitate to reach out again.
